docs/gettingstarted.rst

```diff
@@ -19,14 +19,33 @@ Are you looking at installing and deploying REANA cluster locally on your laptop
 
       $ sudo dpkg -i kubectl*.deb minikube*.deb
 
-2. Start Minikube virtual machine:
+2. Install Helm locally:
+
+   .. code-block:: console
+
+      $ curl https://storage.googleapis.com/kubernetes-helm/helm-v2.12.3-linux-amd64.tar.gz \
```

Comment:
The installation of Helm, happening on host machine, should be the part of item 1 next to minikube and kubectl.
(like you have it amended in user guide actually)

I would probably also use an abstract ```sudo dpkg -i kubernetes-helm*.deb``` similarly to how `kubectl` and `minikube` are being illustrated, with link to upstream installation instructions in the free text.

This is because the concrete method may vary on various GNU/Linux distributions. E.g. on Arch Linux, one uses the AUR package [kubernetes-helm](https://aur.archlinux.org/packages/kubernetes-helm/).  You can add it to the `docs/userguide.rst`.